src/frontend/utils/fagsak.ts

```diff
@@ -11,3 +11,7 @@ export const hentSisteBehandlingPåFagsak = (fagsak: IFagsak): IBehandling | und
         )[0];
     }
 };
+
+export const hentAktivBehandlingPåFagsak = (fagsak: IFagsak): IBehandling | undefined => {
+    return fagsak.behandlinger.find((behandling: IBehandling) => behandling.aktiv === true);
```

Comment:
Ikke noe som trenger å fikses i denne omgangen, men skulle vi endre til at aktiv behandling blir flyttet til en aktivBehandling=behandlingsId i fagsak?

Reply:
Tja, kan potensielt skape trøbbel i fremtiden hvis vi ikke husker å endre det?

Reply:
hmm ja, man må vel huske å endre uansett?  Her kan man vel risikere at man får treff på den første, også ligger det flere aktive, dersom man glemmer å endre. Mens ved å ha en behandlingId i fagsak hvertfall vil sikre at det kun er én og man trenger ikke endre to steder ved oppdatering (sette inaktiv og aktiv). Men denne iden kan jo også være outdated, så blir ikke så stor forskjell :P kjenner ikke koden så godt, så mye mulig det er enklere og bedre løsning å holde styr på å endre status på behandlingen, som her :) Synes bare jeg huska det ble nevnt en gang!

Reply:
Godt poeng! Jeg tror at på sikt vil vi mer aktivt bruke behandlings id'en. F.eks. når man skal gå inn på gamle behandlinger. Så tror ikke vi trenger å gjøre så mye akkurat nå.